.travis.yml

```diff
@@ -1,15 +1,6 @@
 language: java
 jdk: oraclejdk8
 
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-      - sourceline: 'deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8'
-        key_url: 'https://bazel.build/bazel-release.pub.gpg'
-    packages:
-      - bazel
```

Comment:
Indeed!

I was initially concerned that switching from `apt-get` to Bazelisk to download Bazel would cause any caching done by `apt-get` to be eliminated, since Bazelisk downloads Bazel directly from the [GitHub Releases page for Bazel](https://github.com/bazelbuild/bazel/releases), and each time the Bazel sample is run it downloads Bazel afresh. But I now understand from this [StackOverflow question](https://stackoverflow.com/questions/47603077/how-to-cache-packages-which-have-been-installed-via-apt-get-in-travis-ci) that Travis CI doesn't actually cache anything installed with `apt-get` by default. So I'm not concerned that Bazelisk is anymore inefficient than the previous solution.